jonah-dev/lfd2-bot
# Outputs link to launch LFD2 when numbers are reached

## Summary
The goal of this PR is to provide a link in message printed once there are enough ready players in the lobby.

## Testing Instructions
1. Create 8 test accounts and add them to your test server.
2. Run `DISCORD_TOKEN=<your_bot_token> python bot.py`. Your bot and it's token can be created and/or found by following the [tutorial here](https://discordpy.readthedocs.io/en/latest/discord.html).
2. Type the command `?ready` in each account to add 8 players to the lobby.
3. Confirm the message printed after the 8th player is added includes a link to launch LFD2.
4. Click the link and confirm LFD2 is launched.
